packages/page-council/src/Motions/Voting.tsx

```diff
@@ -88,13 +118,32 @@ function Voting ({ hash, idNumber, isDisabled, members, prime, proposal }: Props
                 <div>{t<string>('You are voting with this collective\'s prime account. The vote will be the default outcome in case of any abstentions.')}</div>
               </article>
             )}
+            {willFail && isNewVote && (
+              <article className='warning'>
+                <div>{t<string>('Your "Nay" vote will end this proposal. This will close it.')}</div>
+              </article>
+            )}
+            {willPass && isNewVote && (
+              <article className='warning'>
+                <div>{t<string>('Your "Aye" vote will end this proposal. This will close it.')}</div>
+              </article>
+            )}
+            {!isNewVote && (
+              <article className='warning'>
```

Comment:
```suggestion
              <article className='full warning'>
```

(Just stretches it to the left edge)